docs/proposals/20201123-tracing-ingestion-api-and-opentelemetry-collector.md

```diff
@@ -0,0 +1,89 @@
+# Traces ingestion API and OpenTelemetry collector
+
+* **Owners:**:
+    * `@pavolloffay`
+
+* **Related Tickets:**
+    * [observatorium/issues/442](https://github.com/observatorium/observatorium/issues/442)
+
+* **Other docs:**
```

Comment:
I would just remove this line since there are no further links here

Reply:
done